soc/arm/nordic_nrf/nrf52/dts_fixup.h

```diff
@@ -26,8 +26,16 @@
 #define CONFIG_UART_0_IRQ_NUM		NORDIC_NRF_UART_40002000_IRQ_0
 #define CONFIG_UART_0_BAUD_RATE		NORDIC_NRF_UART_40002000_CURRENT_SPEED
 #define CONFIG_UART_0_NAME		NORDIC_NRF_UART_40002000_LABEL
-#define CONFIG_UART_0_TX_PIN		NORDIC_NRF_UART_40002000_TX_PIN
-#define CONFIG_UART_0_RX_PIN		NORDIC_NRF_UART_40002000_RX_PIN
+  #if defined(NORDIC_NRF_UART_40002000_TX_PIN)
```

Comment:
nack, move the pin cfg to dts, than this isn't needed.

Reply:
I am new to DTS so, would appreciate pointers.

Reply:
Please, check how the pin configuration is provided in dts files in other nRF52-based Board definitions, as well as the dtsi file for nRF52 SOCs.

Reply:
@jpsaman as @ioannisg, just take a look at examples in boards/arm/nrf52840_pca10056/nrf52840_pca10056.dts

Here's the commit that adds uart pins if you want more detail:

commit 7c2197f2d84be4b8728e6480a1c0e6669baff171
Author: Paweł Zadrożniak <pawel.zadrozniak@nordicsemi.no>
Date:   Tue Aug 28 16:24:22 2018 +0200

    boards: nrf: uart: Moved UART pin configuration to DTS (nRF boards)